www/js/banidb/sqlite-search.js

```diff
@@ -68,7 +75,7 @@ const query = (searchQuery, searchType, searchSource) => (
         }
 
         // Use LIKE if anywhere, otherwise use operators
-        if (searchType === CONSTS.SEARCH_TYPES.FIRST_LETTERS_ANYWHERE) {
+        if ((searchType === CONSTS.SEARCH_TYPES.FIRST_LETTERS_ANYWHERE) || isWildChar) {
```

Comment:
By coupling `isWildChar` with `searchType` being first-letter-anywhere, wouldn't this `AGn*` be the same as `*AGn*`?

Reply:
I was thinking about this a bit on the drive this morning and I think @navdeepsinghkhalsa is correct. That doing a a wildcard at the start of a search string for First Letter Start is effectively doing first letter anywhere - the only time the they are different is when it is not the first letter that is the wildcard

Reply:
@navdeepsinghkhalsa I have given various cases in slack, but to answer your this specific question, yes in first letter annywhere, `AGn*` will effectively be `*AGn*`.